[Samples] Products sample error

Products sample error when you upload and publish and try to open products/webapp/index.html

```json
{
	"message": "HTTP request failed",
	"headers": [],
	"statusCode": 404,
	"statusText": "",
	"responseText": "\{\"error\":\{\"code\":null,\"message\":\{\"lang\":\"en-US\",\"value\":\"Could not find an entity set or function import for 'Orders'.\"}}}"
}
```

Logs:
```java
2022-01-25 13:12:01.035 [ERROR] [pool-228-thread-1] c.s.x.hdb.ds.facade.XSKHDBCoreFacade - SAP DBTech JDBC: [329]: cannot use duplicate name of function or procedure: ProductsOrdersCustomerNames: line 1 col 17 (at pos 16)
com.sap.db.jdbc.exceptions.JDBCDriverException: SAP DBTech JDBC: [329]: cannot use duplicate name of function or procedure: ProductsOrdersCustomerNames: line 1 col 17 (at pos 16)
	at com.sap.db.jdbc.exceptions.SQLExceptionSapDB._newInstance(SQLExceptionSapDB.java:209) ~[ngdbc-2.11.14.jar:na]
	at com.sap.db.jdbc.exceptions.SQLExceptionSapDB.newInstance(SQLExceptionSapDB.java:42) ~[ngdbc-2.11.14.jar:na]
	at com.sap.db.jdbc.packet.HReplyPacket._buildExceptionChain(HReplyPacket.java:841) ~[ngdbc-2.11.14.jar:na]
	at com.sap.db.jdbc.packet.HReplyPacket.getSQLExceptionChain(HReplyPacket.java:158) ~[ngdbc-2.11.14.jar:na]
	at com.sap.db.jdbc.packet.HPartInfo.getSQLExceptionChain(HPartInfo.java:39) ~[ngdbc-2.11.14.jar:na]
	at com.sap.db.jdbc.ConnectionSapDB._receive(ConnectionSapDB.java:4786) ~[ngdbc-2.11.14.jar:na]
	at com.sap.db.jdbc.ConnectionSapDB.exchange(ConnectionSapDB.java:2022) ~[ngdbc-2.11.14.jar:na]
	at com.sap.db.jdbc.PreparedStatementSapDB._prepare(PreparedStatementSapDB.java:2961) ~[ngdbc-2.11.14.jar:na]
	at com.sap.db.jdbc.PreparedStatementSapDB._doParse(PreparedStatementSapDB.java:2854) ~[ngdbc-2.11.14.jar:na]
	at com.sap.db.jdbc.PreparedStatementSapDB.<init>(PreparedStatementSapDB.java:166) ~[ngdbc-2.11.14.jar:na]
	at com.sap.db.jdbc.PreparedStatementSapDB9.<init>(Unknown Source) ~[ngdbc-2.11.14.jar:na]
	at com.sap.db.jdbc.HanaPreparedStatement.<init>(Unknown Source) ~[ngdbc-2.11.14.jar:na]
	at com.sap.db.jdbc.HanaPreparedStatementClean.<init>(Unknown Source) ~[ngdbc-2.11.14.jar:na]
	at com.sap.db.jdbc.HanaPreparedStatementClean.newInstance(Unknown Source) ~[ngdbc-2.11.14.jar:na]
	at com.sap.db.jdbc.ConnectionSapDB9._prepareStatement(Unknown Source) ~[ngdbc-2.11.14.jar:na]
	at com.sap.db.jdbc.ConnectionSapDB.prepareStatement(ConnectionSapDB.java:369) ~[ngdbc-2.11.14.jar:na]
	at org.apache.commons.dbcp2.DelegatingConnection.prepareStatement(DelegatingConnection.java:291) ~[commons-dbcp2-2.2.0.jar:2.2.0]
	at org.apache.commons.dbcp2.DelegatingConnection.prepareStatement(DelegatingConnection.java:291) ~[commons-dbcp2-2.2.0.jar:2.2.0]
	at org.eclipse.dirigible.database.api.wrappers.WrappedConnection.prepareStatement(WrappedConnection.java:447) ~[dirigible-database-api-6.1.19.jar:na]
	at com.sap.xsk.hdb.ds.processors.AbstractXSKProcessor.executeSql(AbstractXSKProcessor.java:31) ~[xsk-modules-engines-hdb-0.13.0-SNAPSHOT.jar:na]
	at com.sap.xsk.hdb.ds.processors.hdbtablefunction.HDBTableFunctionCreateProcessor.execute(HDBTableFunctionCreateProcessor.java:54) ~[xsk-modules-engines-hdb-0.13.0-SNAPSHOT.jar:na]
	at com.sap.xsk.hdb.ds.processors.hdbtablefunction.HDBTableFunctionCreateProcessor.execute(HDBTableFunctionCreateProcessor.java:31) ~[xsk-modules-engines-hdb-0.13.0-SNAPSHOT.jar:na]
	at com.sap.xsk.hdb.ds.service.manager.IXSKTableFunctionManagerService.createDataStructure(IXSKTableFunctionManagerService.java:79) ~[xsk-modules-engines-hdb-0.13.0-SNAPSHOT.jar:na]
	at com.sap.xsk.hdb.ds.service.manager.IXSKTableFunctionManagerService.createDataStructure(IXSKTableFunctionManagerService.java:31) ~[xsk-modules-engines-hdb-0.13.0-SNAPSHOT.jar:na]
	at com.sap.xsk.hdb.ds.facade.XSKHDBCoreFacade.updateEntities(XSKHDBCoreFacade.java:498) ~[xsk-modules-engines-hdb-0.13.0-SNAPSHOT.jar:na]
	at com.sap.xsk.hdb.ds.synchronizer.XSKDataStructuresSynchronizer.synchronize(XSKDataStructuresSynchronizer.java:236) ~[xsk-modules-engines-hdb-0.13.0-SNAPSHOT.jar:na]
	at org.eclipse.dirigible.core.scheduler.api.AbstractSynchronizerJob$1.run(AbstractSynchronizerJob.java:52) ~[dirigible-core-scheduler-6.1.19.jar:na]
	at org.eclipse.dirigible.commons.timeout.TimeLimited$1.call(TimeLimited.java:38) ~[dirigible-commons-timeout-6.1.19.jar:na]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[na:na]
	at java.base/java.lang.Thread.run(Thread.java:829) ~[na:na]
```

```java
org.apache.olingo.odata2.api.uri.UriNotMatchingException: Could not find an entity set or function import for 'Orders'.
	at org.apache.olingo.odata2.core.uri.UriParserImpl.handleNormalInitialSegment(UriParserImpl.java:217) ~[olingo-odata2-core-2.0.11.jar:2.0.11]
	at org.apache.olingo.odata2.core.uri.UriParserImpl.handleResourcePath(UriParserImpl.java:187) ~[olingo-odata2-core-2.0.11.jar:2.0.11]
	at org.apache.olingo.odata2.core.uri.UriParserImpl.parseAll(UriParserImpl.java:127) ~[olingo-odata2-core-2.0.11.jar:2.0.11]
	at org.apache.olingo.odata2.core.ODataRequestHandler.handle(ODataRequestHandler.java:108) ~[olingo-odata2-core-2.0.11.jar:2.0.11]
	at org.apache.olingo.odata2.core.rest.ODataSubLocator.handle(ODataSubLocator.java:164) ~[olingo-odata2-core-2.0.11.jar:2.0.11]
	at org.apache.olingo.odata2.core.rest.ODataSubLocator.handleGet(ODataSubLocator.java:58) ~[olingo-odata2-core-2.0.11.jar:2.0.11]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:na]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:na]
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:na]
	at java.base/java.lang.reflect.Method.invoke(Method.java:566) ~[na:na]
	at org.apache.cxf.service.invoker.AbstractInvoker.performInvocation(AbstractInvoker.java:179) ~[cxf-core-3.5.0.jar:3.5.0]
	at org.apache.cxf.service.invoker.AbstractInvoker.invoke(AbstractInvoker.java:96) ~[cxf-core-3.5.0.jar:3.5.0]
	at org.apache.cxf.jaxrs.JAXRSInvoker.invoke(JAXRSInvoker.java:201) ~[cxf-rt-frontend-jaxrs-3.5.0.jar:3.5.0]
	at org.apache.cxf.jaxrs.JAXRSInvoker.invoke(JAXRSInvoker.java:286) ~[cxf-rt-frontend-jaxrs-3.5.0.jar:3.5.0]
	at org.apache.cxf.jaxrs.JAXRSInvoker.invoke(JAXRSInvoker.java:104) ~[cxf-rt-frontend-jaxrs-3.5.0.jar:3.5.0]
	at org.apache.cxf.interceptor.ServiceInvokerInterceptor$1.run(ServiceInvokerInterceptor.java:59) ~[cxf-core-3.5.0.jar:3.5.0]
	at org.apache.cxf.interceptor.ServiceInvokerInterceptor.handleMessage(ServiceInvokerInterceptor.java:96) ~[cxf-core-3.5.0.jar:3.5.0]
	at org.apache.cxf.phase.PhaseInterceptorChain.doIntercept(PhaseInterceptorChain.java:307) ~[cxf-core-3.5.0.jar:3.5.0]
	at org.apache.cxf.transport.ChainInitiationObserver.onMessage(ChainInitiationObserver.java:121) ~[cxf-core-3.5.0.jar:3.5.0]
	at org.apache.cxf.transport.http.AbstractHTTPDestination.invoke(AbstractHTTPDestination.java:265) ~[cxf-rt-transports-http-3.5.0.jar:3.5.0]
	at org.apache.cxf.transport.servlet.ServletController.invokeDestination(ServletController.java:234) ~[cxf-rt-transports-http-3.5.0.jar:3.5.0]
	at org.apache.cxf.transport.servlet.ServletController.invoke(ServletController.java:208) ~[cxf-rt-transports-http-3.5.0.jar:3.5.0]
	at org.apache.cxf.transport.servlet.ServletController.invoke(ServletController.java:160) ~[cxf-rt-transports-http-3.5.0.jar:3.5.0]
	at org.apache.cxf.transport.servlet.CXFNonSpringServlet.invoke(CXFNonSpringServlet.java:225) ~[cxf-rt-transports-http-3.5.0.jar:3.5.0]
	at org.apache.cxf.transport.servlet.AbstractHTTPServlet.handleRequest(AbstractHTTPServlet.java:304) ~[cxf-rt-transports-http-3.5.0.jar:3.5.0]
	at org.apache.cxf.transport.servlet.AbstractHTTPServlet.doGet(AbstractHTTPServlet.java:222) ~[cxf-rt-transports-http-3.5.0.jar:3.5.0]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:634) ~[servlet-api.jar:na]
	at org.apache.cxf.transport.servlet.AbstractHTTPServlet.service(AbstractHTTPServlet.java:279) ~[cxf-rt-transports-http-3.5.0.jar:3.5.0]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) ~[catalina.jar:8.5.43]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:8.5.43]
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52) ~[tomcat-websocket.jar:8.5.43]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:8.5.43]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:8.5.43]
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:728) ~[catalina.jar:8.5.43]
	at org.apache.catalina.core.ApplicationDispatcher.processRequest(ApplicationDispatcher.java:470) ~[catalina.jar:8.5.43]
	at org.apache.catalina.core.ApplicationDispatcher.doForward(ApplicationDispatcher.java:395) ~[catalina.jar:8.5.43]
	at org.apache.catalina.core.ApplicationDispatcher.forward(ApplicationDispatcher.java:316) ~[catalina.jar:8.5.43]
	at com.sap.xsk.xsodata.ds.filter.XSODataForwardFilter.doFilter(XSODataForwardFilter.java:47) ~[xsk-modules-engines-xsodata-0.13.0-SNAPSHOT.jar:na]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:8.5.43]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:8.5.43]
	at org.apache.catalina.filters.CorsFilter.handleNonCORS(CorsFilter.java:364) ~[catalina.jar:8.5.43]
	at org.apache.catalina.filters.CorsFilter.doFilter(CorsFilter.java:170) ~[catalina.jar:8.5.43]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:8.5.43]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:8.5.43]
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:199) ~[catalina.jar:8.5.43]
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:96) ~[catalina.jar:8.5.43]
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:493) ~[catalina.jar:8.5.43]
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:137) ~[catalina.jar:8.5.43]
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:81) ~[catalina.jar:8.5.43]
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:660) ~[catalina.jar:8.5.43]
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:87) ~[catalina.jar:8.5.43]
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343) ~[catalina.jar:8.5.43]
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:798) ~[tomcat-coyote.jar:8.5.43]
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:66) ~[tomcat-coyote.jar:8.5.43]
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:808) ~[tomcat-coyote.jar:8.5.43]
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1498) ~[tomcat-coyote.jar:8.5.43]
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49) ~[tomcat-coyote.jar:8.5.43]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[na:na]
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61) ~[tomcat-util.jar:8.5.43]
	at java.base/java.lang.Thread.run(Thread.java:829) ~[na:na]
```


```java
2022-01-25 13:12:01.063 [ERROR] [pool-230-thread-1] o.e.d.c.s.s.SecuritySynchronizer - Synchronizing process for Privileges and Access artifacts failed.
org.eclipse.dirigible.core.scheduler.api.SynchronizationException: com.sap.xsk.xsaccess.ds.api.XSKAccessException: com.sap.xsk.xsaccess.ds.api.XSKPrivilegeException: No XSK Privilege found with name: {Developer}
	at com.sap.xsk.xsaccess.ds.synchronizer.XSKSecuritySynchronizer.synchronizeAccess(XSKSecuritySynchronizer.java:270) ~[xsk-modules-engines-xssecurity-0.13.0-SNAPSHOT.jar:na]
	at com.sap.xsk.xsaccess.ds.synchronizer.XSKSecuritySynchronizer.processArtifacts(XSKSecuritySynchronizer.java:210) ~[xsk-modules-engines-xssecurity-0.13.0-SNAPSHOT.jar:na]
	at com.sap.xsk.xsaccess.ds.synchronizer.XSKSecuritySynchronizer.synchronize(XSKSecuritySynchronizer.java:102) ~[xsk-modules-engines-xssecurity-0.13.0-SNAPSHOT.jar:na]
	at org.eclipse.dirigible.core.scheduler.api.AbstractSynchronizerJob$1.run(AbstractSynchronizerJob.java:52) ~[dirigible-core-scheduler-6.1.19.jar:na]
	at org.eclipse.dirigible.commons.timeout.TimeLimited$1.call(TimeLimited.java:38) ~[dirigible-commons-timeout-6.1.19.jar:na]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[na:na]
	at java.base/java.lang.Thread.run(Thread.java:829) ~[na:na]
Caused by: com.sap.xsk.xsaccess.ds.api.XSKAccessException: com.sap.xsk.xsaccess.ds.api.XSKPrivilegeException: No XSK Privilege found with name: {Developer}
	at com.sap.xsk.xsaccess.ds.service.XSKAccessCoreService.createXSKAccessDefinition(XSKAccessCoreService.java:79) ~[xsk-modules-engines-xssecurity-0.13.0-SNAPSHOT.jar:na]
	at com.sap.xsk.xsaccess.ds.synchronizer.XSKSecuritySynchronizer.synchronizeAccess(XSKSecuritySynchronizer.java:251) ~[xsk-modules-engines-xssecurity-0.13.0-SNAPSHOT.jar:na]
	... 8 common frames omitted
Caused by: com.sap.xsk.xsaccess.ds.api.XSKPrivilegeException: No XSK Privilege found with name: {Developer}
	at com.sap.xsk.xsaccess.ds.service.XSKAccessCoreService.validatePrivileges(XSKAccessCoreService.java:210) ~[xsk-modules-engines-xssecurity-0.13.0-SNAPSHOT.jar:na]
	at com.sap.xsk.xsaccess.ds.service.XSKAccessCoreService.createXSKAccessDefinition(XSKAccessCoreService.java:61) ~[xsk-modules-engines-xssecurity-0.13.0-SNAPSHOT.jar:na]
	... 9 common frames omitted
```

Related:
- #854
- #1059

--- a/samples/products/db/ProductsOrdersCustomerNames.hdbtablefunction
+++ b/samples/products/db/ProductsOrdersCustomerNames.hdbtablefunction
@@ -1,4 +1,4 @@
-FUNCTION "ProductsOrdersCustomerNames" ( )
+FUNCTION "XSK_SAMPLES_PRODUCTS"."ProductsOrdersCustomerNames" ( )
    RETURNS Table (
        "Id" NVARCHAR(50),
        "CustomerName" NVARCHAR(500)

--- a/samples/products/xsaccess/.xsaccess
+++ b/samples/products/xsaccess/.xsaccess
@@ -5,5 +5,5 @@
             "method": "Form"
          },
   "authorization":                // Privileges for application access
-       ["Developer", "Administrator"]
+       ["products.xsaccess::Developer", "products.xsaccess::Administrator"]
 }

--- a/samples/products/xsjob/OrdersCount.xsjs
+++ b/samples/products/xsjob/OrdersCount.xsjs
@@ -1,7 +1,7 @@
 function countOrders() {
 	var connection = $.db.getConnection();
 	try {
-		var pstmt = connection.prepareStatement('select count(*) from "Products_Orders"');
+		var pstmt = connection.prepareStatement('select count(*) from "products.db::Products.Orders"');
 		var res = pstmt.executeQuery();
 		if (res.next()) {
 			$.trace.info('Count: ' + res.getInteger(1));

